src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -75,13 +75,13 @@ const WorkspaceTabs = ({
 
   const tabs = [
     { name: 'dashboard', link: 'workspace-dashboard' },
-    { name: 'data', link: 'workspace-data' },
-    ...(!isAnalysisTabVisible() ? [{ name: 'notebooks', link: 'workspace-notebooks' }] : []),
+    ...(isGoogleWorkspace ? [{ name: 'data', link: 'workspace-data' }] : []),
```

Comment:
For the initial rollout of Azure workspaces, we are not displaying tabs that depend on Google project IDs/bucket information. As individual teams change their UI to work with Azure workspaces, they can re-enable their tabs.

This is probably the most controversial part of this PR-- it means that all tabs except the Workspace Dashboard won't appear until we have received the Workspace details from the server and know that it is a Google project. This is generally pretty fast (and something that we then cache), and it's not like you can actually do anything useful with the tabs during that time anyway. However, it is a change in behavior so I want to call it out, and I also am going to touch base with Ean in UX to get his opinion.

The other option would be to show all the tabs initially, and then have unsupported ones disappear if we have an Azure workspace. This would not impact existing Google workspaces, but in general I think that it is more jolting for elements to disappear than to appear. And it seems unlikely at this point that the tabs for Azure and Google workspaces will be identical, so there will most likely always be a need for things to appear/disappear.

Reply:
I checked with Ean and he agrees it is better for the tabs to appear than disappear. He did suggest another option of disabling the tabs for Azure, but we came to the conclusion that it may introduce confusion (Azure users aren't necessarily existing users, so having tabs disabled just adds confusion for folks who wouldn't necessarily expect that functionality, plus some of the tabs may never be supported on Azure).

Reply:
Thanks for this discussion. I'm on-board, and we can iterate once we're able to get a feel for it.